File: vintage-kitman API/vintage-kitman API/ViewModels/OrderModels/CustomOrderVM.cs
using vintage_kitman_API.Model;

namespace vintage_kitman_API.ViewModels.OrderModels
{
    public class CustomOrderVM
    {
        public int CustomOrderId { get; set; }
        public string Size { get; set; }
        public string Name { get; set; }
        public int Quantity { get; set; }
        public string Image { get; set; }
        public bool? IsSourcable { get; set; }
        public string? CustomName { get; set; }
        public int? CustomNumber { get; set; }
        public string Id { get; set; }
        //navigation
    }
}

File: vintage-kitman-frontend/src/app/models/orders/custom-order-vm.ts
export interface CustomOrderVM {
    customOrderId: number;
    size: string;
    name: string;
    quantity: number;
    image: string;
    isSourcable: boolean | null;
    customName: string| null;
    customNumber: number | null;
    message: string | null;
  }
  